src/test/java/software/aws/neptune/gremlin/sql/SqlGremlinTest.java

```diff
@@ -75,7 +77,7 @@
 // Temporary test file to do ad hoc testing.
 @Disabled
 public class SqlGremlinTest {
-    private static final String ENDPOINT = "database-1.cluster-cdffsmv2nzf7.us-east-2.neptune.amazonaws.com";
+    private static final String ENDPOINT = "jdbc-bug-bash-iam-instance-1.cdubgfjknn5r.us-east-1.neptune.amazonaws.com";
```

Comment:
Let's have a separate task to remove hardcoded endpoints.

Reply:
Noted.